options/db_options.h

```diff
@@ -89,6 +89,7 @@ struct MutableDBOptions {
   MutableDBOptions();
   explicit MutableDBOptions(const MutableDBOptions& options) = default;
   explicit MutableDBOptions(const DBOptions& options);
+  MutableDBOptions& operator=(const MutableDBOptions& options) = default;
```

Comment:
So why defining a default copy constructor in the first place?

Reply:
Actually, my fix here is not correct. I should've removed the uses of implicit assignment instead.

The `explicit MutableDBOptions(const MutableDBOptions& options)` is probably here to make creating copy of DB's mutable options less convenient, so we don't end up with lots of options with differing values.

Reply:
BTW, the `size_t valid_batches = 0;` is gone in the latest code.